tests/test_bulk_notready.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016-2021 XMOS LIMITED.
```

Comment:
If I run this test, I get 8 passes and 8 failures. The failures are all at 600 MHz core speed, is this to be expected?

Reply:
I had only run smoke tests, thanks for that. Will investigate.

Reply:
Its the FS tests that fail. I have xfailed and will raise an issue.